### PR TITLE
Added examples migrated to Keras 3 in examples_master.py 

### PR DESCRIPTION
**Pull request by dhantule**
I have added examples that have been migrated to Keras 3 in the examples_master.py 

1. Highly accurate boundaries segmentation using BASNet [PR](https://github.com/keras-team/keras-io/pull/2038)
2. Large-scale multi-label text classification [PR](https://github.com/keras-team/keras-io/pull/2052)
3. Multimodal entailment [PR](https://github.com/keras-team/keras-io/pull/2019)
4. Using the Forward-Forward Algorithm for Image Classification [PR](https://github.com/keras-team/keras-io/pull/1932)
5. Approximating non-Function Mappings with Mixture Density Networks [PR](https://github.com/keras-team/keras-io/pull/2050)
6. Classification with Gated Residual and Variable Selection Networks with HyperParameters tuning [PR](https://github.com/keras-team/keras-io/pull/2065)
7. Electroencephalogram Signal Classification for Brain-Computer Interface [PR](https://github.com/keras-team/keras-io/pull/2025)
8. Audio Classification with the STFTSpectrogram layer [PR](https://github.com/keras-team/keras-io/pull/1947)
9. Self-supervised contrastive learning with SimSiam [PR](https://github.com/keras-team/keras-io/pull/1697)

### CODE REVIEW

File: scripts/examples_master.py [hunk 1]
```diff
@@ -423,6 +424,18 @@
...
+                    "title": "Using the Forward-Forward Algorithm for Image Classification",
+                    "subcategory": "Image classification",
+                    "keras_3": True,
+                },
+                {
+                    "path": "simsiam",
+                    "title": "Self-supervised contrastive learning with SimSiam",
+                    "subcategory": "Image similarity search",
+                    "keras_3": True,
+                },
```
Comment (sachinprasadhs) [inline scripts/examples_master.py]: Reorder these under their respective subcategory sections for Image classification and Image similarity search.

File: scripts/examples_master.py [hunk 2]
```diff
@@ -657,6 +672,12 @@
                     "subcategory": "Recommendation",
                     "keras_3": True,
                 },
+                {
+                    "path": "class_with_grn_and_vsn_with_hyperparameters_tuning",
+                    "title": "Classification with Gated Residual and Variable Selection Networks with HyperParameters tuning",
+                    "subcategory": "Structured data classification",
+                    "keras_3": True,
+                },
```
Comment (sachinprasadhs) [inline scripts/examples_master.py]: Same, move it under same subcategory, not mixing with Recommendation.

File: scripts/examples_master.py [hunk 3]
```diff
@@ -710,6 +731,12 @@
                     "subcategory": "Timeseries forecasting",
                     "keras_3": True,
                 },
+                {
+                    "path": "eeg_bci_ssvepformer",
+                    "title": "Electroencephalogram Signal Classification for Brain-Computer Interface",
+                    "subcategory": "Timeseries classification",
+                    "keras_3": True,
+                },
```
Comment (sachinprasadhs) [inline scripts/examples_master.py]: Move with along with Timeseries classification models

File: scripts/examples_master.py [hunk 4]
```diff
@@ -1029,6 +1062,12 @@
                     "subcategory": "ML best practices",
                     "keras_3": True,
                 },
+                {
+                    "path": "approximating_non_function_mappings",
+                    "title": "Approximating non-Function Mappings with Mixture Density Networks",
+                    "subcategory": "Keras usage tips",
+                    "keras_3": True,
+                },
```
Comment (sachinprasadhs) [inline scripts/examples_master.py]: Move it under Keras usage tips

Comment (dhantule) [reply scripts/examples_master.py]: there are 3 subcategories (Keras usage tips, Serving and ML best practices) under Quick Keras Recipes, I have grouped similar categories together

File: scripts/examples_master.py (hunk 2 above)
Comment (dhantule) [reply scripts/examples_master.py]: done

File: scripts/examples_master.py (hunk 4 above)
Comment (sachinprasadhs) [reply scripts/examples_master.py]: Thanks

**Comment by sachinprasadhs** [comment]
Rebase with master to run with latest changes.
